### DISCUSSION TITLE
getting an http images instead of https

### DISCUSSION THREAD

**Discussion by owaaquino** [cloudinary-devs/gatsby-transformer-cloudinary#208] [Q&A]
Currently, I've tried using the gatsby-transformer-cloudinary on my project. Somehow I'm getting an `http` url for my images instead of `https` this makes an issue in my production `blocked:mixedcontent`. Is there any other setup I need to do to make sure I'm getting an `https` for my images? 

This is my graphql query looks like. 

```
const data = useStaticQuery(graphql`
    query {
      cloudinaryAsset(publicId: { eq: "gatsby-cloudinary/event" }) {
        fluid {
          ...CloudinaryAssetFluid
        }
        gatsbyImageData(placeholder: BLURRED)
      }
    }
  `)
```

Then following the plugin documenationtion 

```
const image = getImage(data.cloudinaryAsset) 
```

When I `console.log` this it returns as `http`

<img width="620" alt="Screen Shot 2022-11-30 at 5 42 58 AM" src="https://user-images.githubusercontent.com/12265025/204654338-935d5623-21e3-4437-a7de-2a8188bb7b58.png">

I already tried changing my Allowed fetch domains settings in Cloudinary but no luck. Hope the team can help, as I'm been stuck on this and I wanted to use my cloudinary images.

**Comment by raae** [comment]
Thank you for letting us know. I'll have a look at this today.

**Reply by raae** [reply]
I have been able to reproduce it @owaaquino, will report back when I know more.

**Reply by raae** [reply]
Hopefully fixed in 3.0.1, let me know how it goes @owaaquino.
